<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubAAWidgetClass</class>
 <widget class="QWidget" name="SubAAWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubAAWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="toolPart">
     <item>
      <widget class="QPushButton" name="singleFunctionTestPushButton">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>单模块测试</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/module.svg</normaloff>:/icon/resources/icons/module.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rocketSimulationPushButton">
       <property name="toolTip">
        <string>弹道模拟</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/rocket.svg</normaloff>:/icon/resources/icons/rocket.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="toolTip">
        <string>关闭音乐盒</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none; 
width: 20px; height: 50px;
}
QPushButton:hover {
border: 2px solid blue;
}
QPushButton:pressed {
 border: 2px solid red; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icon/resources/icons/IconamoonCloseCircle1.svg</normaloff>:/icon/resources/icons/IconamoonCloseCircle1.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="singleFunctionWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="airGroupBox">
        <property name="title">
         <string>大气模型</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTableWidget" name="airTableWidget">
           <row>
            <property name="text">
             <string>高度</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>密度</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>大小</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>单位</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>模型</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>m</string>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>kg/m³</string>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>lgρ = 0.088136 - 0.0605317h + 1.37264 × 10⁻⁵h²</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="coordinateGroupBox">
        <property name="title">
         <string>坐标相关</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="TextEditPro" name="eulerRotationParamsTextEdit">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="eulerRotationLabel">
           <property name="text">
            <string>欧拉坐标转换矩阵计算</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="changePushButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/icon/resources/icons/RightArrow.svg</normaloff>:/icon/resources/icons/RightArrow.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QTableWidget" name="eulerResultTableWidget">
           <row>
            <property name="text">
             <string>x</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>y</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>z</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>x</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>y</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>z</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rocketSimulationWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolBox" name="chooseModelToolBox">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="airModelPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>113</width>
           <height>704</height>
          </rect>
         </property>
         <attribute name="label">
          <string>大气模型</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>113</width>
           <height>704</height>
          </rect>
         </property>
         <attribute name="label">
          <string>积分方法</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mainSimPart">
        <item>
         <widget class="RocketSimOpenGLWidget" name="openGLWidget">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextEditPro</class>
   <extends>QTextEdit</extends>
   <header>customize/texteditpro.h</header>
  </customwidget>
  <customwidget>
   <class>RocketSimOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>AACustomizeClass/rocketsimopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
